hooks/_common.sh

```diff
@@ -445,6 +445,46 @@ function common::colorify {
   echo -e "${COLOR}${TEXT}${RESET}" >&2
 }
 
+#######################################################################
+# Set terraform binary path
+# Allows user to set the path to custom Terraform or OpenTofu binaries
+# Outputs:
+#   If failed - exit 1 with error message about missing Terraform/Opentofu binary
+function common::get_tf_binary {
+  local hook_config_tf_binary
+
+  for config in "${HOOK_CONFIG[@]}"; do
+    if [[ $config == tf_binary=* ]]; then
+      hook_config_tf_binary="${config#*=}"
+      break
+    fi
+  done
+
+  # direct hook config, has the highest priority
+  if [[ -n "${hook_config_tf_binary}" ]]; then
+    echo "${hook_config_tf_binary}"
+
+  # environment variable
+  elif [[ -n "${PCT_TFPATH}" ]]; then
+    echo "${PCT_TFPATH}"
+
+  # Maybe there is a similar setting for Terragrunt already
+  elif [[ -n "${TERRAGRUNT_TFPATH}" ]]; then
+    echo "${TERRAGRUNT_TFPATH}"
+
+  # check if Terraform binary is available
+  elif command -v terraform >/dev/null 2>&1; then
```

Comment:
For the sake of brevity
```suggestion
  elif command -v terraform &>/dev/null; then
```